test/util/util.go

```diff
@@ -265,7 +284,24 @@ func SensorPodLogContains(ctx context.Context, kubeClient kubernetes.Interface,
 		return false, fmt.Errorf("error getting sensor pod name: %w", err)
 	}
 	podName := podList.Items[0].GetName()
-	fmt.Printf("Sensor POD name: %s", podName)
+	if len(podList.Items) > 1 {
```

Comment:
I think you can simplify the code by removing this guard and the outcome should be the same?